### ISSUE TITLE
Perform sanitisation within the library 

### ISSUE THREAD

**Issue by Andrew-Murtagh** [fetchableai/fetchable-python#3]
**Is your feature request related to a problem? Please describe.**
Takes additional effort on behalf of the client to sanitize the inputs, i.e. to transform multiple, and potentially capitalised words, into snake case. E.g.
```
client.fetch_entity_attribute("Empire State building", "Height")
```
should be entered as:
```
client.fetch_entity_attribute("empire_state_building", "height")
```

**Describe the solution you'd like**
Create a private helper function in the client FetchableClient class which transforms any input into the correct format before making a request to the backend.

**Describe alternatives you've considered**
Do not create helper function and do the transformation in each function, i.e. within the 'fetch_entity_attribute' function - or any other functions.

**Additional context**
N/A

**Comment by Andrew-Murtagh** [comment]
commit 5eb8249cc8ac1c8f1c08af206ce4479abb8b599d closes #3

**Comment by Andrew-Murtagh** [comment]
commit 5eb8249cc8ac1c8f1c08af206ce4479abb8b599d closes #3
